catalog/ui/src/app/util.ts

```diff
@@ -78,6 +78,7 @@ export function recursiveAssign(target: object, source: object): any {
 type RenderContentOpt = {
   allowIFrame?: boolean;
   format?: 'asciidoc' | 'html';
+  attributes?: object;
```

Comment:
Perhaps for future compatibility with other template formats we should call this `vars` or `variables`? rather than the asciidoc specific `attributes`? The asciidoc naming seems a little odd anyway.